apps/accounts/tests/test_views.py

```diff
@@ -109,6 +111,92 @@ def test_two_users_exist(self):
             views.find_user({'uid': '9999', 'email': 'you@amo.ca'})
 
 
+class TestRenderErrorHTML(TestCase):
+
+    def make_request(self):
+        return APIRequestFactory().get(reverse('accounts.authenticate'))
+
+    def login_error_url(self, **params):
+        return urlparams(reverse('users.login'), **params)
+
+    def render_error(self, error, next_path=None):
+        return views.render_error(
+            self.make_request(), error, format='html', next_path=next_path)
+
+    def test_error_no_code(self):
```

Comment:
I don't understand why you need to define five tests here. It only looks like you need a couple cases:
````
render_error(<any error code>)
assert response['location'] == self.login_error_url(error=<any error code>)
````
then another case for passing through the destination URL if requested.